editor/src/components/editor/store/reparent-target.ts

```diff
@@ -63,3 +65,57 @@ export function reparentTargetToString<P extends ElementPath>(
     return EP.toString(reparentTargetParent)
   }
 }
+
+export function commonReparentTarget(
+  first: ReparentTargetParent<ElementPath>,
+  second: ReparentTargetParent<ElementPath>,
+): ReparentTargetParent<ElementPath> | null {
+  if (reparentTargetParentIsElementPath(first)) {
+    if (reparentTargetParentIsElementPath(second)) {
+      return EP.closestSharedAncestor(first, second, true)
+    } else {
+      return EP.closestSharedAncestor(first, second.elementPath, true)
+    }
+  } else {
+    if (reparentTargetParentIsElementPath(second)) {
+      return EP.closestSharedAncestor(first.elementPath, second, true)
+    } else {
+      if (EP.pathsEqual(first.elementPath, second.elementPath)) {
+        if (first.clause === second.clause) {
+          return first
+        } else {
+          return first.elementPath
+        }
```

Comment:
Can you add a comment in here please because although it's clear what is being returned, it's not clear why that is being returned